.gitignore

```diff
@@ -15,6 +15,7 @@ wp-tests-config.php
 /tests/phpunit/build
 /wp-cli.local.yml
 /jsdoc
+php.log
```

Comment:
Let's remove this, same as for any OS- or editor-specific files you should add it to your global gitignore instead.

Reply:
No, you missed the change in `tests/phpunit/includes/bootstrap.php`; because the code is using `error_log()` the message gets written to the console unless a log file is defined. The messages mess up the unit test output, so I defined a default log file and excluded it from git.

Reply:
Ok... this doesn't belong in the PR for `CP_STRICT`.

I also use inline `error_log` pretty often when working on tests, so this is worth discussing separately.